tasks/configure.yml

```diff
@@ -42,3 +43,15 @@
   ansible.builtin.file:
     mode: 0700
     path: "{{ task_conf_file }}"
+
+- name: Stop service {{ task_conf_service }}
+  ansible.builtin.service:
+    name: "{{ task_conf_service }}"
+    state: stopped
+  register: service_status
+
+- name: Restart service if it was started
+  ansible.builtin.service:
+    name: "{{ task_conf_service }}"
+    state: started
+  when: service_status.changed
```

Comment:
This should be done in a handler instead.  You can probably just `notify` the handler in `tasks.main.yml` if and only if `configure.yml` actually modifies the configuration file in question.

Reply:
resolved with [4d1c132](https://github.com/cisagov/ansible-role-clamav/pull/75/commits/4d1c1324f032a5b8451f456c1053382e55f4c9e0) commit, using handlers